src/buffer.cc

```diff
@@ -340,7 +340,7 @@ Result FormatBuffer::CopyData(const std::vector<Value>& data) {
       continue;
     }
 
-    const auto& components = format_->GetComponents();
+    const auto components = format_->GetComponents();
```

Comment:
Why not keep this a const ref?

Reply:
Accidentally dropped. I'd changed GetComponents but things went sideways and I changed it back and missed this one ....

Reply:
done